tests/testthat/test-independent-testing-rtf_assemble.R

```diff
@@ -1,106 +1,86 @@
 # test bulletproofing
+file <- replicate(2, tempfile(fileext = ".rtf"))
```

Comment:
I prefer each test to be stand-alone, but this works well and the test file is not so sprawling that we lose track.